packages/tsconfig/docs/examples.md

```diff
@@ -2,72 +2,45 @@
 
 ## Getting started
 
-This project outputs multiple tsconfig.json files for use. If you are unsure which one to use, this guide exists to help you understand which one is most applicable to your project.
+This project outputs a single tsconfig.json file for use. This file is configured to disable all output options (`declaration` is `false`, `noEmit` is true) and is set to strictly type check your code (for example, `noImplicitAny` and many other strict type checks). It is likely that you will need to import this file and do a little configuration additionally depending on whether you plan to use this configuration in conjunction with Webpack or Babel, or whether you plan to use `tsc` to output artifacts.
 
 ### Questions to answer
 
 1. Do you have a full TypeScript project (e.g. is your project brand new or is it sufficiently small that you can convert the whole thing at once)?
-2. Do you need to keep using Babel to transpile an existing project, or can you use tsc (the TypeScript compiler that is installed when you add `typescript` as a Node dependency) to keep your build simple?
+2. Do you need to keep using Babel to transpile an existing project, or can you use `tsc` (the TypeScript compiler that is installed when you add `typescript` as a Node dependency) to keep your build simple?
 3. Is your project a library, which means it needs to compile in a way that other TypeScript projects can use the types from it, or is it an app, which means it likely has JSX and that there is no need to export declarations?
 
 ### Overriding options
 
-Use the `"extends"` option in your tsconfig.json to extend the appropriate config. You may override an option (that is the nature of extendable configs) but try to avoid doing so as it means more maintenance for you and your team later!
+Use the `"extends"` option in your tsconfig.json to extend the appropriate config.
 
 ## Apps
 
-This setup assumes that create-react-app, webpack, and/or babel will transpile your bundled assets (aka `tsc` is not used for emitting transpiled code, only for type-checking).
+This setup assumes that Create React App, Next, Webpack, and/or Babel will transpile your bundled assets (aka `tsc` is not used for emitting transpiled code, only for type-checking).
+
+Extend your `tsconfig.json`:
 
-Extend your `tsconfig.json` with our `tsconfig.app.json`
 ```json
 {
-  "extends": "@spotify/tsconfig/tsconfig.app.json",
-  "include": ["src"]
+  "extends": "@spotify/tsconfig"
```

Comment:
this still needs `include: ["src"]`, doesn't it?

Reply:
It's pretty dependent on how you use the TypeScript config - for most of the Webpack-based ways it's not strictly necessary.